### PR TITLE
Changed all get calls to the WiW API to include '?include_objects=fal…

### PR DESCRIPTION
**Pull request by jrauschenberg**
#### What's this PR do?
It changes get calls to the WiW API to narrow down the information being requested. For instance, if only shift information is needed, the call now says "?include_objects=false" so that user and location information doesn't also come along in the response. It will reduce the overall usage of the WiW API.
#### Where should the reviewer start?
Look at the differences on github or do a global find for WiW API get calls.
#### How should this be manually tested?
Look at each modified WiW API call and make sure that it is of the appropriate scope. You can also use "node runJobs" to make sure the modified calls have not broken the code.
#### What are the relevant tickets?
[INT-51](https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-51)
#### Questions:

…se' if appropriate. Will cut down on API use.

### CODE REVIEW

**Comment by mbogor** [comment]
this branch has merge conflicts that have to be resolved

File: test/index.js [hunk 1]
```diff
@@ -8,16 +8,16 @@ global.KEYS = require('../keys.js');
...
+// var retrieveAndRenderShiftsToDelete = require('./retrieveAndRenderShiftsToDelete.js');
+// var whenIWorkUnofficialAPI = require('./whenIWorkUnofficialAPI.js');
+// var timezone = require('./timezone.js');
+// var deleteShiftsAndRedirect = require('./deleteShiftsAndRedirect.js');
+// var renderShiftDeleteSuccess = require('./renderShiftDeleteSuccess.js');
+// var mergeOpenShifts = require('./mergeOpenShifts.js');
+// var notifyMoreShifts = require('./notifyMoreShifts.js');
+// var timeOffRequests = require('./timeOffRequests');
+// var recurShifts = require('./recurShifts.js');
+// var updateCanvas = require('./updateCanvas');
```
Comment (mbogor) [inline test/index.js]: uncomment tests

File: jobs/scheduling/base.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+global.KEYS = require('../../keys.js')
```
Comment (mbogor) [inline jobs/scheduling/base.js]: delete before committing

File: jobs/scheduling/NotifyMoreShifts.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ function notifyMoreShifts() {
     location_id: CONFIG.locationID.regular_shifts
   };
 
-  wIWUserAPI.get('shifts', query, function (response) {
+  wIWUserAPI.get('shifts?include_objects=users', query, function (response) {
```
Comment (mbogor) [inline jobs/scheduling/NotifyMoreShifts.js]: this messes with the api call so that no users are returned as a response

**Comment by mbogor** [comment]
👍

**Comment by tongxiang** [comment]
Begun this convo in slack, but: 


Instead of modifying 17 files, what are your thoughts on just modifying the WiW API `get` request?

https://github.com/CrisisTextLine/Elmer/pull/154/files

Are there any obvious downsides?

**Comment by jrauschenberg** [comment]
Per Tong/John conversation, modify WiW unofficial API instead of changing all GET calls.

**Comment by jrauschenberg** [comment]
Merged in a slightly modified version as we discussed!

**Pull request merged**